src/main/java/org/alfresco/repo/bulkimport/metadataloaders/XmlPropertiesFileMetadataLoader.java

```diff
@@ -97,6 +100,15 @@
 {
     private final static Log log = LogFactory.getLog(XmlPropertiesFileMetadataLoader.class);
     private final static String METADATA_FILE_EXTENSION = "properties.xml";
+    // MNT-18001

```

Comment:
The comments on the private member could instead be added as javadoc comments on the public setter.